dll/win32/rsaenh/rsaenh.c

```diff
@@ -1104,7 +1105,7 @@ static BOOL create_container_key(KEYCONTAINER *pKeyContainer, REGSAM sam, HKEY *
     CHAR szRSABase[MAX_PATH];
     HKEY hRootKey;
 
-    sprintf(szRSABase, RSAENH_REGKEY, pKeyContainer->szName);
+    StringCbPrintfA(szRSABase, sizeof(szRSABase), RSAENH_REGKEY, pKeyContainer->szName);
```

Comment:
That's Wine code. You should surround it with `#if __REACTOS__`

Reply:
I think I already explained it. There shouldn't be any need to use that. That's not how version control works. "Syncing" is not copy the new code over the old and manually review for __REACTOS__. That is retarded. Git can do that much better.

Reply:
I understand there's a better way in theory... but I don't think that changes anything unless we actually implement that way?

Reply:
Well I guess someone has to start it. Oh wait... I already started it! But if that was not visible enough for everyone, I might as well go ahead full on radical progressive and reorganize the codebase. Otherwise things won't move...
I hope I scared some people now 😆